### DISCUSSION TITLE
Can this software be used for AIMD？

### DISCUSSION THREAD

**Discussion by JoesonCheung** [deepmodeling/reacnetgenerator#1704] [Q&A] [answered]
I have obtained a trajectory with CP2K. Can I use reacnetgenerator to generate a reaction network? How should I do it? Which type should I choose?

**Comment by njzjz** [comment]
One can use `--type xyz` to process XYZ format.

**Reply by JoesonCheung** [reply]
Dr. Zeng, thanks for your reply. The attachment is the trajectory that I got by AIMD. How could I  use the command of reacnetgenerator? I've tried the command `reacnetgenerator --type xyz -i test.xyz -a Si Cl H --nohmm` . But it doesn't work. 
[test.zip](https://github.com/tongzhugroup/reacnetgenerator/files/9593900/test.zip)

**Reply by njzjz** [reply]
The current supported XYZ file format is something like below
[![image](https://user-images.githubusercontent.com/9496702/190930104-a112f440-6272-427e-90c9-ccb2ed62b7d1.png)](https://en.wikipedia.org/wiki/XYZ_file_format#Example)

**Reply by JoesonCheung** [reply]
Can you show me an example of the command line for type xyz?  Thank you so much.

**Reply by njzjz** [reply, marked as answer]
A very simple example is [here](https://github.com/tongzhugroup/reacnetgenerator/blob/master/tests/inputs/water.xyz).

You can try `reacnetgenerator -i water.xyz --type xyz --nohmm -a O H` and it should not throw any errors. Note that it is just for testing. As it only contains one frame, it will not give anything.
